packages/cadl-ranch-specs/http/client/structure/default/mockapi.ts

```diff
@@ -3,22 +3,22 @@ import { ScenarioMockApi } from "@azure-tools/cadl-ranch-api";
 
 export const Scenarios: Record<string, ScenarioMockApi> = {};
 Scenarios.Client_Structure_Service = passOnSuccess([
-  mockapi.patch("/one", (req) => {
+  mockapi.patch("client/structure/default/one", (req) => {
```

Comment:
```suggestion
  mockapi.patch("/client/structure/default/one", (req) => {
```

think they should start with /